spec.html

```diff
@@ -32078,8 +32130,10 @@ <h1>Atom</h1>
         <emu-alg>
           1. Evaluate |Disjunction| with argument _direction_ to obtain a Matcher _m_.
           1. Let _parenIndex_ be the number of left-capturing parentheses in the entire regular expression that occur to the left of this |Atom|. This is the total number of <emu-grammar>Atom :: `(` GroupSpecifier Disjunction `)`</emu-grammar> Parse Nodes prior to or enclosing this |Atom|.
-          1. Return an internal Matcher closure that takes two arguments, a State _x_ and a Continuation _c_, and performs the following steps:
-            1. Let _d_ be an internal Continuation closure that takes one State argument _y_ and performs the following steps:
+          1. Return a new Matcher with parameters (_x_, _c_) that captures _m_ and _parenIndex_ and performs the following steps when called:
+            1. Assert: _x_ is a State.
+            1. Assert: _c_ is a Continuation.
+            1. Let _d_ be a new Continuation with parameters (_y_) that captures _c_ and performs the following steps when called:
               1. Let _cap_ be a copy of _y_'s _captures_ List.
```

Comment:
Is there a reason this one doesn't have the `Assert: _y_ is a State` bit?

Reply:
Nope. Will add.